src/ui/connect-dashboard-ui/src/FamilyHubs.RequestForSupport.Web/appsettings.json

```diff
@@ -87,7 +87,8 @@
       "CookieName": "connect_family_support_cookies_policy",
       "CookieVersion": 1,
       "MeasurementId": "",
-      "ContainerId": ""
+      "ContainerId": "",
+      "ClarityId": "lwxqmeq8no"
```

Comment:
Do we want a Clarity Id in appsettings?
Would that mean we sent data to Clarity from our local dev machines?
Would it make more sense to have it blank in appsettings and just populate it in the envs?

Reply:
Yes ^

We have the appsettings as `"ClarityId": ""` and then set the Clarity ID in KeyVault for both Test & Prod.

Test is `s181t01-kv-fh-general` and the name of the secret to add will be `CONNECT-DASHBOARD-UI-FAMILYHUBSUI-ANALYTICS-CLARITYID`

Same name for Prod (with its respective value obvs), the name of that KeyVault is `s181p01-kv-fh-general`

On the next pipeline deployment of `release-clarity-content` it will then pick it up :)

Reply:
@lazcool 
> Would that mean we sent data to Clarity from our local dev machines?

If you're working on clarity you absolutely want it to be set locally.

FWIW this was just copy-pasted from https://github.com/DFE-Digital/fh-services/blob/main/src/ui/connect-ui/src/FamilyHubs.Referral.Web/appsettings.json#L108
I realise on this branch we've actually removed the default values so I'll update this to match.